Sources/ValCommand/ValCommand.swift

```diff
@@ -84,6 +105,20 @@ public struct ValCommand: ParsableCommand {
       valueName: "output-type"))
   private var outputType: OutputType = .binary
 
+  @Option(
+    name: [.customLong("cc")],
+    help: ArgumentHelp(
+      "Customize the CXX compiler used by the Val backend. From: clang, gcc, msvc(Windows only)",
+      valueName: "CXXCompiler"))
+  private var customizeCXXCompiler: CXXCompiler = .clang
+
+  @Option(
+    name: [.customLong("cc-flags")],
+    help: ArgumentHelp(
+      "Specify flags for the CXX compiler to use",
+      valueName: "CXXCompilerFlags"))
+  private var ccFlags: [String] = ["Empty"]
```

Comment:
Why the need for a particular value and not simple:
```suggestion
  private var ccFlags: [String] = []
```

Reply:
Sorry, my fault